examples/lessons/06/app.js

```diff
@@ -44,6 +44,7 @@ var filters = ['nearest', 'linear', 'mipmap'];
 const animationLoop = new AnimationLoop({
   // .context(() => createGLContext({canvas: 'lesson05-canvas'}))
   onInitialize: ({canvas, gl}) => {
+    addControls();
 
     addEvents(canvas, {
```

Comment:
Moved addEvents to seperate method 'addKeyboardHandler()' , same as lesson7, if that is what you are referring to.